v63003/outref/gtm8735.txt

```diff
@@ -24,8 +24,39 @@ Database file ##TEST_PATH##/mumps.dat now has BG access method
 %YDB-W-WCWRNNOTCHG, Not all specified database files were changed
   Access method                          BG  Global Buffers                1024
   LOCK shares DB critical section     FALSE  Read Only                      OFF
-# Displaying status of gtmhelp database
+# Displaying status of gtmhelp databases
```

Comment:
Add at the end of the line : "to verify files have read-only permissions for user/group/other"